homework/04LessonFour.js

```diff
@@ -87,7 +109,7 @@ var nestedObjects = {
 // What is an associative array? What is the difference between an array and an associative array?
 //
 // Your Answer Goes Here:
-//
+// An associative array is a collection of of data called a key, the key has an index that is assigned specific values. Arrays have an index that start at specifically at 0.
```

Comment:
Your mind is in the right place here. An associative array is a collection of named keys that each contain data/value.